plugins/inputs/elasticsearch_query/elasticsearch_query.go

```diff
@@ -173,6 +173,12 @@ func (e *ElasticsearchQuery) connectToES() error {
 	return nil
 }
 
+// This is here to allow us to conform to the ServiceInput interface, which is
+// required for Stop() to be called.
```

Comment:
Same here and for all instances below. Please remove all those comments as it should be clear for someone familiar with Telegraf's plugin code.